packages/core/crates/core-packet/src/interaction.rs

```diff
@@ -367,7 +367,10 @@ impl<Db: HoprCoreEthereumDbActions> AcknowledgementInteraction<Db> {
                 let response = unackowledged.get_response(&ack.ack_key_share)?;
                 debug!("acknowledging ticket using response {}", response.to_hex());
 
-                let ack_ticket = AcknowledgedTicket::new(unackowledged.ticket, response, unackowledged.signer);
+                let domain_separator = self.db.read().await.get_channels_domain_separator().await.unwrap();
```

Comment:
Can we handle the error instead of `unwrap` ?

Reply:
obsolete due to refactoring